package com.coderio.task.shared.exception;

public class InternalErrorException extends RuntimeException {

}
